### ISSUE TITLE
Inconsistencies with concepts related to taxes, allowances and charges

### ISSUE THREAD

**Issue by idolum** [OP-TED/ePO#722]
Several concepts were added to descripe taxes, allowances and charges imposed on an item which have an impact on the price of the item:

(1a) There are two concepts describing taxes as part of the price: epo-ord:TaxInformation and epo-ord:ChargeInformation
* epo-ord:TaxInformation defined as "Information about the imposition of mandatory levies required by law."
* epo-ord:ChargeInformation defined as "Information about tax, fee or duty imposed.", the additional information explicitely mentions VAT

Proposed change: Either consolidate classes or exclude taxes from epo-ord:ChargeInformation information.

(1b) Similar to this: epo:isCalculatedOn is defined "The monetary amount to which the multiplier factor is applied in calculating the amount of this allowance or charge.", but only used at epo-ord:TaxInformation

(2) The additional information for epo-cat:ChargeInformation says: "The charge can be fixed or relative to the price." But there is only an attribute epo-cat:hasPercentage allowing only expression relative charges, not fixed amounts.

Proposed change: Add a association to MonetaryValue like epo-cat:hasAmount as it is at epo-ord:TaxInformation.

**Comment by andreea-pasare** [comment]
The following modifications are proposed for ePO 5.0.0:
(1a) modified definition for epo-cat:ChargeInformation: 
```
 "Information about fee or duty imposed.

Additional Information:
Charge category indicates the nature of the duty/fee, for example CAR, etc.
Charge category modifier may be used in case different levels, exemptions or other modifications apply.
The charge can be fixed or relative to the price."
```

(1b) modified definition for `epo:isClaculatedOn` to include tax since it is used at both `epo-ord:TaxInformation` and `epo-ord:AllowanceChargeInformation` levels:
`"The monetary amount to which the multiplier factor is applied in calculating the amount of this allowance, charge or tax."`

(2) the `epo-cat:hasAmount` is inherited from the parent class `epo-ord:AllowanceChargeInformation`:

![Image](https://github.com/user-attachments/assets/9fbae88c-2255-44d9-81b8-63e3fd7c5cc9)

**Comment by andreea-pasare** [comment]
@idolum, does this implementation cover your needs?

**Comment by idolum** [comment]
@andreea-pasare Yes! I aligned the catalogue transactions accordingly. Thank you!
